var/spack/repos/builtin/packages/py-colossalai/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyColossalai(PythonPackage):
+    """An integrated large-scale model training system with efficient
+    parallelization techniques."""
+
+    homepage = "https://www.colossalai.org/"
+    pypi     = "colossalai/colossalai-0.1.0.tar.gz"
```

Comment:
```suggestion
    pypi     = "colossalai/colossalai-0.1.3.tar.gz"
```
This only matters for `spack url summary`